port_drayage_plugin/src/port_drayage_worker.cpp

```diff
@@ -82,8 +82,28 @@ namespace port_drayage_plugin
         using boost::property_tree::ptree;
         ptree pt;
         pt.put("cmv_id", _cmv_id);
-        pt.put("cargo_id", _cargo_id);
         pt.put("operation", PORT_DRAYAGE_ARRIVAL_OPERATION_ID);
+
+        // Add current vehicle location (latitude and longitude)
+        ptree location;
+        location.put("latitude", _current_gps_position.latitude * 10000000); // Convert degrees to 1/10 microdegrees
+        location.put("longitude", _current_gps_position.longitude * 10000000); // Convert degrees to 1/10 microdegrees
```

Comment:
Is there a reason to encode degrees vs 1/10 microdegrees? Json has support for floats, so I think it'd be all the same under-the-hood, no? 1/10th microdegrees lines up nicely with J2735, but every application I've ever seen interact with that data from J2735 immediately converts back to degrees for usability. Is floating point precision the concern?

Reply:
The only reason it was decided to encode latitude and longitude values as 1/10 microdegrees was to line up with the J2735 specification. I am fine with encoding as degrees, and will make sure the change is applied on the V2XHub side as well.